cypress/integration/registration-error-validation-test.js

```diff
@@ -0,0 +1,107 @@
+
+describe("Error validations on Registration page should be visible ", function () {
+  const baseUrl = Cypress.env("baseUrl");
+  const viewportWidth = Cypress.env("viewportWidth");
+  const viewportHeight = Cypress.env("viewportHeight");
+
+  const registerLink = "#register-link";
+  const nameBlankId = "#registration-name-blank";
+  const emailInvalidId = "#registration-email-invalid";
+  const emailBlankId = "#registration-email-blank";
+  const passwordBlankId = "#registration-password-blank";
+  const passwordDuplicateId = "#registration-password-duplicate";
+  const nameInput = 'input[name="name"]';
+  const emailInput = 'input[name="emaial"]';
+  const password1Id = "#pw1";
+  const password2Id = "#pw2";
+  const submitButton = ".button";
+  const invalidEmailList = [
+    "#@%^%#$@#$ @#.com",
+    "@example.com",
+    "Joe Smith < email@example.com>",
+    "email.example.com",
+    "email@example @example.com",
+    ".email @example.com",
+    "email.@example.com",
+    "email..email@example.com",
+    "あいうえお@example.com",
+    "email@example.com(Joe Smith)",
+    "email@example",
+    "email@-example.com",
+    "email@example.web",
+    "email@111.222.333.44444",
+    "email@example..com",
+    "Abc..123@example.com"
+  ]
+
+
+  const emailValid = "j@k.com";
```

Comment:
same thing as the invalid emails, "j@k.com" doesn't represent good coverage of valid emails.

There's some here: https://gist.github.com/cjaoude/fd9910626629b53c4d25
And some here: https://www.w3resource.com/javascript/form/email-validation.php

Reply:
Provided a new list of valid emails!